core/src/main/resources/static/js/modifyEnvs.js

```diff
@@ -158,7 +158,19 @@ app.controller("modifyEnvsCtrl", function($scope, $http, $location, $window) {
                     data: {'envSelected' : envId, 'envType' : envType}
                 }).success(function(output) {
                     if(output != null && output !== ""){
+
                         $scope.envDetails = output;
+
+                        $scope.envDetails.topicRegex = output.params.topicRegex[0];
+                        $scope.envDetails.topicSuffix = output.params.topicSuffix[0];
+                        $scope.envDetails.topicPrefix = output.params.topicPrefix[0];
+
+                        $scope.envDetails.defaultPartitions = output.params.defaultPartitions[0];
```

Comment:
am able to save prefix, suffix, and regex on an env. like below

Topic Prefix : test1
Topic Suffix : test2
Topic Regex : test3

Reply:
Yes that was a conscious decision, what determines what is used is if the advanced topic Configuration is selected or not.
But it allows them to trial the regex and if needed switch back with the slide of a slider in the modification environment page..